lib/scope.rb

```diff
@@ -26,9 +26,14 @@ def self.all_scopes
       @scopes
     end
 
-    def self.class_for(scope_name)
+    def self.initialize_scope(scope_name, json, scope_file_store)
```

Comment:
Seems we are not handling this consistently. `Renderer::for` instantiates the object, `Inspector::for` returns the class.

What about a naming convention of using `class_for` to return the class, and `for` to return an instantiated object?